src/components/uploaded-file/uploaded-file.tsx

```diff
@@ -0,0 +1,61 @@
+import { trpc } from "@/client/lib/trpc";
+import { IconButton, Skeleton } from "@mui/material";
+import { useEffect } from "react";
+import styles from "./uploaded-file.module.css";
+import { QuestionMark } from "@mui/icons-material";
+import DeleteIcon from "@mui/icons-material/Delete";
+
+interface UploadedFileProps {
+  fileId: string;
+  onDelete?: () => void;
+}
+
+export const UploadedFile = ({ fileId, onDelete }: UploadedFileProps) => {
+  const { data, mutate } =
+    trpc.file.getPresignedUrlForFileDownload.useMutation();
+
+  useEffect(() => {
+    mutate({ file_id: fileId });
+  }, [mutate, fileId]);
+
+  return (
+    <div className={styles.container}>
+      {data ? (
+        <>
+          {data.content_type.startsWith("image/") ? (
+            <img src={data.url} className={styles.image} />
```

Comment:
Does the `<img>` need an alt tag?

Also, is there a difference in using `<img>` here instead of `<Image>` from next?

Reply:
Alt could just be "User-uploaded image". Also the next image uses some optimizations to make things faster (can't say exactly what though).

Reply:
sure, will add an alt

> Also, is there a difference in using <img> here instead of <Image> from next?

in our case using `next/image` would likely make things slower as it can take 1-5s upon first load to transform and cache the result (especially because we're not using Vercel's optimized transform pipeline)

it's useful for public images or ones that are shown to many users, but I think just using `<img/>` is likely the better experience in this case

Reply:
1de9bc2